openebs/openebs-provisioner.go

```diff
@@ -240,3 +248,13 @@ func main() {
 	}
 
 }
+
+// GetPersistentVolumeClass returns StorageClassName.
+func GetPersistentVolumeClass(options controller.VolumeOptions) *string {
+	// Use beta annotation first
+	if class, found := options.Parameters[BetaStorageClassAnnotation]; found {
```

Comment:
I have a query here. Are annotations & parameters same here? Are these annotations of PVC? Are these parameters of SC? Are these annotations of SC?

Reply:
those are volume provisioning parameters from StorageClass

Reply:
However, we are interested to check for annotations right from PVC. One of annotation will have the name of SC. I guess the logic is wrong.

Reply:
changed to options.PVC